api/v1alpha1/clusterextension_types.go

```diff
@@ -121,11 +121,6 @@ func init() {
 
 // ClusterExtensionStatus defines the observed state of ClusterExtension
 type ClusterExtensionStatus struct {
-	// +optional
-	InstalledBundle *BundleMetadata `json:"installedBundle,omitempty"`
-	// +optional
-	ResolvedBundle *BundleMetadata `json:"resolvedBundle,omitempty"`
-
```

Comment:
These should stay. If `BundleMetadata` is defined in `extension_types.go`, let's move those definitions to this file.

Reply:
Done.